src/libs/Fullstory/index.native.ts

```diff
@@ -1,13 +1,32 @@
 import FullStory, {FSPage} from '@fullstory/react-native';
-import type {OnyxEntry} from 'react-native-onyx';
+import {type OnyxEntry} from 'react-native-onyx';
+import {useOnyx} from 'react-native-onyx';
 import * as Environment from '@src/libs/Environment/Environment';
 import type {UserMetadata} from '@src/types/onyx';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
 
 /**
  * Fullstory React-Native lib adapter
  * Proxy function calls to React-Native lib
  * */
 const FS = {
+    /**
+     * Initializes FullStory
+     */
+    init: () => {
+        Environment.getEnvironment().then((envName: string) => {
+            // We only want to start fullstory if the app is running in
+            // production
+            if(envName !== CONST.ENVIRONMENT.PRODUCTION) {
```

Comment:
NAB, since linter doesn't seem to mind 
```suggestion
            if (envName !== CONST.ENVIRONMENT.PRODUCTION) {
```